src/l500/l500-color.cpp

```diff
@@ -493,10 +493,44 @@ namespace librealsense
         return tags;
     }
 
-    std::vector<uint8_t> l500_color::get_raw_intrinsics_table() const
+    ivcam2::intrinsic_rgb l500_color::get_intrinsics_table() const
     {
-        AC_LOG( DEBUG, "RGB_INTRINSIC_GET" );
-        return _hw_monitor->send(command{ RGB_INTRINSIC_GET });
+        // Get RAW data from firmware
```

Comment:
Please keep the AC_LOG as it makes it clear when actual FW commands are sent and when the cache is used.

Reply:
But not sure that AC will be the first to get to this function right?
It will write it on AC log even if it wasn't called from AC.

Reply:
Added